cmd/zt_scenario_helpers_for_test.go

```diff
@@ -345,6 +345,18 @@ func (scenarioHelper) generateBlobsFromList(a *assert.Assertions, containerClien
 	time.Sleep(time.Millisecond * 1050)
 }
 
+func (scenarioHelper) generateVersionsForBlobsFromList(a *assert.Assertions, containerClient *container.Client, blobList []string) {
```

Comment:
This doesn't operate too differently from generateBlobsFromList. Furthermore, we should generate some semblance of actually random data here instead, that way when generating multiple versions it can be proven that each version actually differs(?)

Reply:
Updated this method to allow additional of customized strings for data to allow variation of random data which can be set in the test itself.